src/mmw/js/src/core/views.js

```diff
@@ -124,11 +124,17 @@ var HeaderView = Marionette.ItemView.extend({
                     window.location.replace(cloneUrl);
                 })
                 .fail(function() {
-                    window.alert(
-                        'There was an error trying to clone that project. ' +
-                        'Please ensure that the project ID is valid and ' +
-                        'that you have permission to view the project.'
-                    );
+                    var alertView = new modalViews.AlertView({
+                        model: new modalModels.AlertModel({
+                            alertMessage:'There was an error trying to clone that project. ' +
+                                    'Please ensure that the project ID is valid and ' +
+                                    'that you have permission to view the project.',
+                            alertType:'Error',
+                            dismissLabel:'Okay'
```

Comment:
Since the `dismissLabel` is _always_ `'Okay'`, consider giving it a default value in `AlertModel` and only overriding it when different.